docs/source/terms_and_model.rst

```diff
@@ -698,3 +703,60 @@ subclasses, but are still treated as variation.
 
 .. _GitHub issue: https://github.com/ga4gh/vr-spec/issues
 .. _genetic variation: https://en.wikipedia.org/wiki/Genetic_variation
+
+
+Variation Set
```

Comment:
Should this be VariationSet (without spaces, so that same as SequenceLocation headers

Reply:
Yep. Another good catch.